src/MSBuild/LiveLogger/LiveLogger.cs

```diff
@@ -311,15 +317,15 @@ public void Shutdown()
             if (succeeded)
             {
                 Console.WriteLine(ANSIBuilder.Formatting.Color("Build succeeded.", ANSIBuilder.Formatting.ForegroundColor.Green));
-                Console.WriteLine($"\t{warningCount} Warning(s)");
-                Console.WriteLine($"\t{errorCount} Error(s)");
             }
             else
             {
                 Console.WriteLine(ANSIBuilder.Formatting.Color("Build failed.", ANSIBuilder.Formatting.ForegroundColor.Red));
-                Console.WriteLine($"\t{warningCount} Warnings(s)");
-                Console.WriteLine($"\t{errorCount} Errors(s)");
             }
+            Console.WriteLine($"\t{warningCount} Warnings(s)");
+            Console.WriteLine($"\t{errorCount} Errors(s)");
+            Console.WriteLine();
+            Console.WriteLine($"Time elapsed {buildDuration.ToString()}");
```

Comment:
I definitely want this rounded down, nobody cares about their build time to the ten-thousandth of a second :)

But you got me thinking, how do we feel about something like this instead?

![image](https://user-images.githubusercontent.com/3347530/219503315-b1b39029-89e7-47c3-8ae3-3122e2a852ee.png)

Reply:
I'm good with that 🙂

Reply:
So you mean only minutes:seconds and not fractions of a second?